eng/targets/ResolveReferences.targets

```diff
@@ -227,10 +227,6 @@
       Condition=" '@(PackageReference->HasMetadata('Private')->Count())' != '0' "
       Code="BUILD004"
       Text="%25(Private) metadata should not be applied to the %(Identity) package reference. Did you mean %25(PrivateAssets)?" />
-    <Warning
```

Comment:
Is this just not true anymore?

Reply:
That warning was wrong to begin with. The `PrivateAssets` metadata is supported on both ProjectReference and PackageReference. This started failing when building with desktop msbuild.